src/pybind/mgr/dashboard/module.py

```diff
@@ -976,14 +995,15 @@ def index(self):
                 }
 
                 return template.render(
+                    url_prefix = global_instance().url_prefix,
                     ceph_version=global_instance().version,
                     path_info='/osd' + cherrypy.request.path_info,
                     toplevel_data=json.dumps(toplevel_data, indent=2),
                     content_data=json.dumps(content_data, indent=2)
                 )
 
-        cherrypy.tree.mount(Root(), "/", conf)
```

Comment:
This part confuses me: if I have a prefix of "/foo" and one is using a reverse proxy under /foo, isn't cherrypy's "/" going to appear as /foo anyway?

So by adding the prefix in the `.mount` call, doesn't this end up serving at `/foo/foo/`?